webrtc.html

```diff
@@ -8811,6 +8811,30 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     </section>
     <section>
       <h4><dfn>RTCDTMFSender</dfn></h4>
+        <p>To create an <code>RTCDTMFSender</code>, the user agent MUST
+        run the following steps:</p>
+        <ol>
+          <li>
+            <p>Let <var>dtmf</var> be a newly created
+            <code><a>RTCDTMFSender</a></code> object.</p>
+          </li>
+          <li>
+            <p>Let <var>dtmf</var> have a <dfn>[[\CanInsertDtmf]]</dfn>
```

Comment:
I don't see where this slot is set to true. Is an internal slot necessary/helpful for this?